guides/common/modules/proc_importing-into-the-library-environment.adoc

```diff
@@ -39,7 +36,8 @@ total 68M
 --path=/var/lib/pulp/imports/2021-03-02T03-35-24-00-00
 ----
 +
-Note you must enter the full path `/var/lib/pulp/imports/<path>`. Relative paths do not work.
+Note you must enter the full path `/var/lib/pulp/imports/_My_Exported_Library_Dir_`.
+Relative paths do not work.
 . To verify that you imported the Library content, check the contents of the Product and Repositories.
 A new Content View called `Import-Library` is created in the target organization.
-This Content View is used to facilitate the library content import.
+This Content View is used to facilitate the Library content import.
```

Comment:
Should add some more info here:

"By default, this content view is not shown in the web UI.
'Import-Library' is __not__ meant to be assigned directly to hosts. Instead, assign your hosts to 'Default Organization View' or another content view as you would normally."